docs/web/examples/index.rst

```diff
@@ -9,8 +9,9 @@ Examples
 twisted.web.client
 ------------------
 
-- :download:`getpage.py` - use ``twisted.web.client.getPage`` to download a web page.
-- :download:`dlpage.py` - add callbacks to ``twisted.web.client.downloadPage`` to display errors that occur when downloading a web page
+- :download:`httpclient.py` - use ``twisted.web.client.Agent`` to download a web page.
+- (deprecated) :download:`getpage.py` - use ``twisted.web.client.getPage`` to download a web page.
+- (deprecated) :download:`dlpage.py` - add callbacks to ``twisted.web.client.downloadPage`` to display errors that occur when downloading a web page
```

Comment:
The (possibly unwritten) policy is that documentation sticks around as long as the code does. People inheriting a code base that is using the deprecated functionality, and are assiduously updating and cleaning up `DeprecationWarnings` may need to look at the documentation to see what the deprecated things are doing, to figure out how to replace them.